src/config/dynamic-theme-fixes.config

```diff
@@ -1021,9 +1021,53 @@ IGNORE IMAGE ANALYSIS
 app.grammarly.com
 
 CSS
+use.valueCircle_f6otssy {
+    stroke: url(#pb-gradient-0);
+}
 [class*="-alerts-markSelectedHigh"], span[class*="markSelectedFocused"] {
     color: rgb(14, 16, 26) !important;
 }
+._c0560d9f-paidview-counter ._a7150ba2-paidview-counterContent {
+    fill: rgb(0, 0, 0, 105);
+}
+@keyframes f15bts9f {
+    to {
+        background-color: rgb(255, 219, 227, 0.25)
+    }
+}
+@keyframes f2hfcpk {
+    to {
+        background-color: rgb(255, 241, 212, 0.25)
+    }
+}
+@keyframes fzka3gx {
+    to {
+        background-color: rgb(219, 252, 255, 0.25)
+    }
+}
+@keyframes f97acta {
+    to {
+        background-color: rgb(242, 224, 255, 0.25)
+    }
+}
+@keyframes fd3zwmz {
+    to {
+        background-color: rgb(240, 242, 252, 0.25)
+    }
+}
+@keyframes f1gz3vdc {
+    to {
+        background-color: rgb(219, 235, 255, 0.25)
+    }
+}
+@keyframes fjzahul {
+    to {
+        background-color: rgb(215, 247, 240, 0.25)
+    }
+}
```

Comment:
These all use obscure naming, not sure if it should be added.

Reply:
That's all on the part of Grammarly there. Those are the names for the different animated highlights. The only change I made was turning the opacity down to 25% because, at 100%, the text was near-impossible to see at pure white.

Reply:
The problem with such naming is that they are automatically generated and when grammarly pushes out another update(small or big) those naming will be gone.

Reply:
The question is then, how do I modify those colors?

Reply:
Not really, keyframes are a edge-case within this config.